doc/contributors-guide/contributing-guidelines.md

```diff
@@ -1,5 +1,13 @@
 # ℹ Contributing guidelines
 
+{% hint style="info" %}
+Hi there wave Thank you for your interest to contribute to the [Git and Github training resource](https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/contributing-guidelines).
+Find the guidelines for contributing to the training resource in the links below.
+{% endhint %}
+
+https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/contributing-guidelines
+
```

Comment:
Hi @LucianeFernandesRoque
Nice.

Please do these:
- Remove the link you added directly under the link
- Use emoji symbol for wave instead.